ui/apps/platform/src/Containers/Policies/Wizard/policyValidationSchemas.ts

```diff
@@ -17,8 +17,15 @@ import { policySectionValidators } from './Step3/policyCriteriaValidators';
 
 type PolicyStep1 = Pick<ClientPolicy, 'name' | 'severity' | 'categories'>;
 
+const policyNameLengthMessage = 'Policy name must be between 5 and 128 characters';
 const validationSchemaStep1: yup.ObjectSchema<PolicyStep1> = yup.object().shape({
-    name: yup.string().trim().required('Policy name is required'),
+    name: yup
+        .string()
+        .trim()
+        .min(5, policyNameLengthMessage)
+        .max(128, policyNameLengthMessage)
+        .matches(/^[^\n$]*$/, 'Policy name must not contain newlines or dollar signs')
```

Comment:
Pro and con for our future selves of comment with backend filename that is source of truth for validation?

central/policy/service/validator.go